docs/self-managed/backup-restore/zeebe-backup-and-restore.md

```diff
@@ -0,0 +1,157 @@
+                "brokerVersion": "8.1.0"
+            },
+            "failureReason": null,
+            "createdAt": "2022-09-26T14:57:17.914044812Z",
+            "lastUpdatedAt": "2022-09-26T14:57:19.107670926Z"
+        },
+        {
+            "partitionId": 2,
+            "status": "COMPLETED",
+            "description": {
+                "snapshotId": "18-12-7-8",
+                "checkpointPosition": 9,
+                "brokerId": 0,
+                "brokerVersion": "8.1.0"
+            },
+            "failureReason": null,
+            "createdAt": "2022-09-26T14:57:17.914367447Z",
+            "lastUpdatedAt": "2022-09-26T14:57:18.887925972Z"
+        }
+    ]
+}
+
+```
+
+`status` gives the overall status of the backup. It is
+
+- `COMPLETED` if all partitions have completed the backup
+- `FAILED` if at least one partition have failed. In this case `failureReason` contains a string describing the reason for failure.
+- `DOES_NOT_EXIST` if at least one partition's backup does not exist
+- `IN_PROGRESS` if at least one partition's backup is in progress
+
+## Restore
+
+A new Zeebe cluster can be created from a specific backup. We provide a standalone app which must be run on each node where a zeebe broker will be running. This is a Spring Boot application similar to the broker and can run using the binary provided as part of the distribution. The app can be configured the same way a broker is configured - via environment variables or using the configuration file located in `config/application.yaml`.
+
+To restore a zeebe cluster, run the following in each node where the broker will be running.
+
+```
+tar -xzf zeebe-distribution-X.Y.Z.tar.gz -C zeebe/
+./bin/restore <backupId>
+```
+
+:::note
+When restoring, provide the same configuration (node id, data directory, cluster size and replication count) as for the broker that will be running in this node. The partition count must be same as in the backup.
+:::
+
+## Configuration
+
+- Enable backups by setting the flag ZEEBE_BROKER_EXPERIMENTAL_FEATURES_ENABLEBACKUP to `true`.
+- The backup management api is available via management port of the gateway. Ensure the configuration `MANAGEMENT_ENDPOINTS_WEB_EXPOSURE_INCLUDE` include "backups".
+- Backup is stored in an external storage. This must be configured before starting the Zeebe cluster.
+
+Configure backup store in the configuration file as follows.
+
+```
+zeebe:
+  broker:
+    data:
+      backup:
+        store: S3
+          s3:
+            bucketName:
+            region:
+            endpoint:
+            accessKey:
+            secretKey:
+```
+
+Alternatively, you can configure backup store using environment variables.
+
+- `ZEEBE_BROKER_DATA_BACKUP_STORE` - Currently, only S3 is supported. You can use any S3 compatible storage.
+- `ZEEBE_BROKER_DATA_BACKUP_S3_BUCKETNAME` - The backup will be stored in this bucket. The bucket must already exist. The bucket must not be shared with other Zeebe clusters.
```

Comment:
Maybe worth clarifying that it _is_ okay (and expected) to share one bucket across all brokers within a cluster.